docs/wiki/development/reading-files.md

```diff
@@ -39,7 +39,7 @@ In order to internalize the main API, consider the same example without error ch
 int main(int argc, char* argv[]) {
   std::string content;
   osquery::readFile("/foo/bar.txt", content);
-  std::cout << "Contents of " << kPath << ":\n" << content;
+  std::cout << "Contents of " << "/foo/bar.txt" << ":\n" << content;
```

Comment:
Nice catch, however, if this is being changed into a string, all of those stream operators are no longer necessary. Consider:

```cpp
std::cout << "Contents of /foo/bar.txt:\n" << content;
```